spec/function.dd

```diff
@@ -1803,6 +1785,53 @@ void venus()
 }
 ---
 
+$(H4 $(LNAME2 struct-return-methods, Struct Return Methods))
+
+        $(P Struct non-static methods marked with the `return` attribute ensure the returned
+        reference will not outlive the struct instance.
+        )
+
```

Comment:
I think this should explicitly clarify that `return` means making the implicitly `ref` this pointer `return ref`.